iml-services/iml-device/tests/integration_tests.rs

```diff
@@ -0,0 +1,144 @@
+// Copyright (c) 2020 DDN. All rights reserved.
+// Use of this source code is governed by a MIT-style
+// license that can be found in the LICENSE file.
+
+extern crate iml_device;
+
+use chrono::{DateTime, Utc};
+use device_types::{
+    mount::{FsType, Mount, MountOpts, MountPoint},
+    DevicePath,
+};
+use dotenv::dotenv;
+use iml_device::{get_db_pool, update_client_mounts};
+use iml_wire_types::Fqdn;
+use insta::assert_json_snapshot;
+use std::error::Error;
+
+#[derive(serde::Serialize)]
+struct ClientMount {
+    id: i32,
+    state_modified_at: DateTime<Utc>,
+    state: String,
+    immutable_state: bool,
+    not_deleted: Option<bool>,
+    content_type_id: Option<i32>,
+    filesystem: String,
+    host_id: i32,
+    mountpoints: Vec<String>,
+}
+
+#[tokio::test]
+#[ignore = "Requires an active and populated DB"]
```

Comment:
Wouldn't it better belong to the tests crate?

Reply:
Tests crate?

Reply:
Well, like `iml-system-docker-tests`

Reply:
This test is meant to sit between unit tests and system tests (integration test). I can run this test in 30 seconds instead of 2 hours once I have a populated db.